docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/configuration-reference/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Lake Loader configuration reference"
+sidebar_position: 1
+---
+
+```mdx-code-block
+import {versions} from '@site/src/componentVersions';
+import Tabs from '@theme/Tabs';
+import CodeBlock from '@theme/CodeBlock';
+import DeltaConfig from '@site/docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/configuration-reference/_delta_config.md';
+import PubsubConfig from '@site/docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/configuration-reference/_pubsub_config.md';
+import KafkaConfig from '@site/docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/configuration-reference/_kafka_config.md';
+import CommonConfig from '@site/docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/configuration-reference/_common_config.md';
+```
+
+<p>The configuration reference in this page is written for Lake Loader <code>{`${versions.lakeLoader}`}</code></p>
+
+### Table configuration
+
+<Tabs groupId="format" queryString>
```

Comment:
```suggestion
<Tabs groupId="lake-format" queryString>
```

A bit more descriptive so that we can use it on other pages throughout the docs without conflicts